Tests/Common/Statistics/AnnualPerformanceTests.cs

```diff
@@ -0,0 +1,134 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+using System.Linq;
+using System.Collections.Generic;
+using NUnit.Framework;
+using QuantConnect.Util;
+using System;
+using QuantConnect.Data.Market;
+using System.Globalization;
+
+namespace QuantConnect.Tests.Common.Statistics
+{
+    [TestFixture]
+    public class AnnualPerformanceTests
+    {
+        private List<TradeBar> _spy = new List<TradeBar>();
+
+        [SetUp]
+        public void GetSPY()
+        {
+            var symbol = Symbol.Create("SPY", SecurityType.Equity, Market.USA);
+            var path = LeanData.GenerateZipFilePath(Globals.DataFolder, symbol, new DateTime(2020, 3, 1),Resolution.Daily, TickType.Trade);
+            foreach (var line in QuantConnect.Compression.ReadLines(path))
+            {
+                var elements = line.Split(',').ToList();
+                var format = "yyyyMMdd HH:mm";
+                var bar = new TradeBar(DateTime.ParseExact(elements[0], format, CultureInfo.InvariantCulture),
```

Comment:
Nice 👍 
I think it would be better to use `TradeBar.ParseEquity()` directly, it would save all the parsing and having to know how to do it too! Maybe we could push this to a static method and reuse in these different tests, let's chat offline